README.md

```diff
@@ -2,28 +2,26 @@
 
 ![](/images/OpenDEX_Full_Transparent.png)
 
-**OpenDEX is a community maintaining the OpenDEX protocol specifications** called **BOLD** (for Basics of L2 DEX). It is not a company or any other sort of legal entity - solely a community of motivated people maintaining some documents on GitHub. All implementations following BOLD, together comprise the OpenDEX network, a network for unstoppable P2P trading. OpenDEX was started because we believe people should start agreeing on one protocol, so that there can be **one global trading network** and not hundreds. Find the slides from the first public announcement at #hcpp19 in Prague [here](201901005_hcpp19.pdf) and the video recording [here](https://www.youtube.com/watch?v=euSr9A6tI90).
+**OpenDEX is a global network for unstoppable P2P Trading of cryptocurrencies and tokens.** At its core, it is a set of open protocol specifications, called **BOLD** (for Basics of L2 DEX). These specifications ensure that OpenDEX implementations remain compatible and users of different software implementations can trade with each other. **OpenDEX is not a company** or any other sort of legal entity, it is an open community of motivated individuals maintaining the BOLD specifications on GitHub. All node implementations following BOLD form the OpenDEX network. OpenDEX was started, because we believe there should be only one DEX, **one global trading network**; not hundreds. Find the slides from the first public announcement at #hcpp19 in Prague [here](201901005_hcpp19.pdf) and the video recording [here](https://www.youtube.com/watch?v=euSr9A6tI90).
```

Comment:
are we really a L2 Dex? I would think it's L3 since it's Bitcoin + Lightning/Raiden + Xud

Reply:
True that.

Reply:
also the lightning specs use "basis" rather than "basics", in case we want to conform

Reply:
"OpenDEX was started because we believe there should be only one DEX, **one global trading network**, not hundreds."

Reply:
Fixed.

Reply:
Forgot to push. Now.